### ISSUE TITLE
Liquid Separation

### ISSUE THREAD

**Issue by unknown-4o4** [studio-minus/ppg-bugs-and-requests#1580]
**Feature description**
A device which allows you to specifically remove/extract liquids from a container (Flasks, Syringes, etc)

**Feature purpose**
Very helpful for creating very intricate chemicals with 100% potency/strength (Such as 100% Exotic Fluid)

**Additional context**
The UI Idea I quickly made: 
![liquid separator](https://user-images.githubusercontent.com/81907908/176891167-03d01032-a305-457d-914f-157e5668283f.png)

🤓☝☝ #62861.2871 repost  (reddit mod wannabe, god damn. we're just trying to help make the game a little more enjoyable.)

**Comment by DindinYT37** [comment]
It's manual which is not very fitting to people playground, otherwise could be interesting. But this is a duplicate of #1263, though this request has more details.

**Comment by DindinYT37** [comment]
> I do feel like this would fit well in the game when dealing with liquids

The fact that it's manual doesn't though.

**Comment by PossiblyRemember** [comment]
> > I do feel like this would fit well in the game when dealing with liquids
> 
> The fact that it's manual doesn't though.

True, people playground is good at making most of the work automatic

**Comment by unknown-4o4** [comment]
> It's manual which is not very fitting to people playground, otherwise could be interesting. But this is a duplicate of #1263, though this request has more details, is better explained, and generally the idea seems more refined and polished.

It was just a proof of concept, I'm not asking for the system to be implemented using my exact idea; I just want the concept to be thought about.

**Comment by DindinYT37** [comment]
> > It's manual which is not very fitting to people playground, otherwise could be interesting. But this is a duplicate of #1263, though this request has more details, is better explained, and generally the idea seems more refined and polished.
> 
> It was just a proof of concept, I'm not asking for the system to be implemented using my exact idea; I just want the concept to be thought about.

I know, I don't hate on you for your idea - I'm just saying that it doesn't fit and if zooi reads it that it's something to be acknowledged.
